helm/prometheus-meta-operator/templates/prometheus-rules/prometheus.rules.yml

```diff
@@ -22,3 +22,16 @@ spec:
         severity: page
         team: atlas
         topic: observability
+    ## Pages Atlas when prometheus fails to send samples to cortex
+    - alert: PrometheusFailsToCommunicateWithRemoteStorageAPI
+      annotations:
+        description: '{{`Prometheus can''t communicate with Remote Storage API.`}}'
+      expr: rate(prometheus_remote_storage_samples_failed_total{app="prometheus"}[15m]) > 0.25
+      for: 15m
```

Comment:
This is purely for my own understanding, if we have `rate(...[15m])` that means the alert will go to _PENDING_ when error rate was >= 25% over the last 15 minutes, and with `for: 15m` it means it will go to _FIRING_ if this keeps up for another 15 minutes. Is that right? (Just checking as I'm not sure how this works exactly)

Reply:
Technically, rate(...[15m]) has nothing to do with error rate. It will average the metric over 15 minutes so the alert will fire if for 15 minutes, the rate for 15 minutes is over 25%. 

As tthe rate over 25 minutes is an average, it will change over time.

You can see an example here:
![image](https://user-images.githubusercontent.com/2787548/113837512-37aa7400-978e-11eb-926e-f72f9de4d2e2.png)

Reply:
It appeared during testing that we need to rate over 1 hour due to dropped samples
![image](https://user-images.githubusercontent.com/2787548/113837764-73ddd480-978e-11eb-9558-964ecddf4c60.png)


As you can see in peu01, when remote write is working, this rate is at 0:
![image](https://user-images.githubusercontent.com/2787548/113837891-8d7f1c00-978e-11eb-99ed-b5d5210c9aa1.png)

Reply:
Ack. I was asking about the interaction between the range `[15m]` and `for: 15m` setting on the alert, as it's one of those things that I'm not quite sure of how it works.

So when there's no samples the `rate()` is 0? Wouldn't that be OK though? As in if there's no samples that probably means Prometheus is having other issues and there's other alerts to cover that case?

Reply:
Technically, prometheus should always send data to cortex as this is another process as this is always scraping. If this is 0, as you say, this is another issue. We could alert on lack of send samples total as well

Reply:
@mig4 do you mind taking another look?